### PR TITLE
docs: migration guide more info

### PR DESCRIPTION
**Pull request by DanRibbens**

### CODE REVIEW

File: docs/migration-guide/overview.mdx [hunk 1]
```diff
@@ -21,14 +21,90 @@ All breaking changes are listed below. If you encounter changes that are not exp
...
+    ```bash
+      pnpm i next react react-dom payload @payloadcms/ui @payloadcms/next
+    ```
+
+    Also install the other @payloadcms packages specific to the plugins and adapters you are using. Depending on your project, these may include:
+      - @payloadcms/db-mongodb
+      - @payloadcms/db-postgres
+      - @payloadcms/richtext-slate
+      - @payloadcms/richtext-lexical
+      - @payloadcms/plugin-form-builder
```
Comment (jacobsfletch) [inline docs/migration-guide/overview.mdx:43]: Many of these plugin don't need to be listed here bc they have not changed from 2.0->3.0

File: docs/migration-guide/overview.mdx [hunk 2]
```diff
@@ -21,14 +21,90 @@ All breaking changes are listed below. If you encounter changes that are not exp
...
+      - @payloadcms/plugin-stripe
+      - @payloadcms/plugin-cloud-storage - Read [More](#@payloadcms/plugin-cloud-storage).
+
+1. Uninstall deprecated packages:
+
+    ```bash
+    pnpm remove express nodemon @payloadcms/bundler-webpack @payloadcms/bundler-vite
+    ```
+
+1. For Payload Cloud users, the plugin has changed.
```
Comment (jacobsfletch) [inline docs/migration-guide/overview.mdx:59]: Plugins are listed at the bottom of this doc

File: docs/migration-guide/overview.mdx [hunk 3]
```diff
@@ -21,14 +21,90 @@ All breaking changes are listed below. If you encounter changes that are not exp
...
 ```bash
   npx create-payload-app@beta
 ```
 
 For more details, see the [Documentation](https://payloadcms.com/docs/getting-started/installation).
 
+1. **Install new dependencies of payload, next.js and react**:
+
+    ```bash
+      pnpm i next react react-dom payload @payloadcms/ui @payloadcms/next
```
Comment (jacobsfletch) [inline docs/migration-guide/overview.mdx:35]: These were likely installed already through cpa which is listed in the previous bullet

Comment (DanRibbens) [reply docs/migration-guide/overview.mdx:35]: The point of listing them is if they're updating all the versions, we pin versions now so this mattters.

The idea that they should just run CPA and then copy files to me is really incomplete as to how they should update. It also is not very clear.

Comment (jacobsfletch) [reply docs/migration-guide/overview.mdx:35]: Not advocating against this, just need to be as clear as possible

**Pull request merged**